decidim-admin/app/controllers/decidim/admin/conflicts_controller.rb

```diff
@@ -5,8 +5,10 @@ module Admin
     class ConflictsController < Decidim::Admin::ApplicationController
       layout "decidim/admin/users"
 
+      before_action :users_current_organization, only: :index
+
       def index
-        @conflicts = Decidim::Verifications::Conflict.all
+        @conflicts = Decidim::Verifications::Conflict.where(current_user: @organization_users)
```

Comment:
```suggestion
        @conflicts = Decidim::Verifications::Conflict.where(current_user: current_organization_users_ids)
```

Also, do you know why the table has `current_user` and `managed_user` fields? should we filter by both?

Reply:
When you try to impersonate another user through verification, you're current user and the impersonate user is managed user.